src/contentScript/context.ts

```diff
@@ -45,6 +45,7 @@
 
 export function markReady(): void {
   ready = true;
+  (window as any).pbReady = true;
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/pixiebrix/pixiebrix-extension/pullRequest?prid=7684115)
```suggestion
  (window as unknown).pbReady = true;
```